src/javascript.rs

```diff
@@ -121,6 +126,39 @@ impl<'a> Generator<'a> {
         }
     }
 
+    fn import(
+        &mut self,
+        module: &'a Vec<String>,
+        as_name: &'a Option<String>,
+        unqualified: &'a Vec<UnqualifiedImport>,
+    ) -> Document<'a> {
+        let module_name = as_name
+            .as_ref()
+            .map(|n| n.as_str().to_doc())
+            .unwrap_or_else(|| Document::String(module.join("_")));
+        let path = Document::String(module.join("/")).surround("\"./", ".js\"");
```

Comment:
I think this will generate invalid code for modules that are nested when imports.

i.e. If `one/two` imports `three/four` then the path will resolve to `one/two/three/four`

Reply:
Updated to now correctly generate path `../three/four`. I'm leaving all package imports to handle as a single thing relating to issue #1072 

For now I think this works for all cases of handling imports from code generated in this project.

(Note I also think this will work for gleam packages dependend on for now because they all become generated modules in the gen dir)